### ISSUE TITLE
Investigate if date inputs become gray on Mac Safari; if so, make it darker

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1541]
Google a11y testers reported that dates (only in the past?) inputted into `<input type="date">` fields on macOS Safari render as a light gray, which doesn't have enough contrast to meet a11y standards. 

![image](https://user-images.githubusercontent.com/77128518/124015413-a0327700-d9b2-11eb-9011-ba731a6f0669.png)

I haven't been able to reproduce on Chrome, and I can't see why this would be happening, but it would be great if someone could try to reproduce the issue.

* Is this only on Mac?
* Is this only for dates in the past?
* Is this only using screen readers?

If we can reproduce this light-gray text problem and can identify when it occurs, let's change it to use darker text that has a contrast ratio of 4.5:1 or greater (as per a11y guidelines).

## Tester's original report

> ENVIRONMENT:
OS | Browser | Screen Reader
macOS Big Sur:11.4| Safari Version 14.1.1 (16611.2.7.1.4)
Tool Used: Color contrast Analyser
 
> STEPS TO REPRODUCE:
1) Browse the above URL on Safari.
2) Linear navigate to 'Continue' button and activate.
3) Linear navigate to submit button and activate.
4) Linear navigate to the 'program for question lifecycle' program and activate the 'Apply' button.
5) Linear navigate to the 'Continue' button and activate.
6) Add backdated value for the date (The numeric value will be light gray)
7) Validate the color contrast of the numeric date value.

> ACTUAL RESULT: Date value (light gray on white) has insufficient color
the contrast of 2.4:1

> EXPECTED RESULT: Color contrast ratio for regular size text should be at least 4.5:1

**Comment by ndmckinley** [comment]
Doesn't reproduce on safari on mac.

<img width="335" alt="Screen Shot 2021-06-30 at 1 26 09 PM" src="https://user-images.githubusercontent.com/605324/124026641-cc47fb00-d9a6-11eb-969f-9a98f2602321.png">
